src/screens/scanning/ConfirmValueScreen.tsx

```diff
@@ -75,13 +68,22 @@ export default function ConfirmValueScreen({
   return (
     <SafeArea>
       <StandardHeader>
-        {voucherMap.size === 0 ? (
-          <StandardLogo />
-        ) : (
+        {/* <TouchableOpacity onPress={() => navigation.navigate('ReviewScreen')}>
           <VoucherCounter>
             <CounterText>{voucherMap.size}</CounterText>
           </VoucherCounter>
-        )}
+        </TouchableOpacity> */}
+        <TouchableOpacity
+          onPress={() =>
+            handlePreventLeave({
+              hasUnsavedChanges,
+              navigation,
+              dispatch,
+            })
+          }
+        >
+          <Icon name="close" size={24} color={Colors.midBlack} />
+        </TouchableOpacity>
```

Comment:
Should this button just return the user to the page they were just on? As of now, it exits the entire voucher entry flow. There may be instances where a user feels unsure about the serial number / barcode they just inputted.

Reply:
Great catch, this makes a lot of sense.